src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import {
+  FC, useEffect, useRef, useContext,
+} from 'react';
+import classNames from 'classnames';
+import { TodoContext } from './TodoContext';
+
+export const ErrorNotification: FC = () => {
+  const {
+    error,
+    onErrorChange,
+  } = useContext(TodoContext);
+
+  const errorMessage = useRef(error);
```

Comment:
try to use `error` only. no need to add ref

Reply:
without using ref I have the text of the error disappears before the error itself and therefore there is a sharp jump when the text disappears, as it disappears smoothly, and with the help of ref I solve this problem)

Reply:
you can leave like this, but try also to add conditional rendering of `ErrorNotification` component in `TodoApp` (render only if there is error message)